isolate_example/lib/page_three.dart

```diff
@@ -103,25 +106,27 @@ class DataTransferIsolateController extends ChangeNotifier {
 
   void generateOnSecondaryIsolate() {
     if (running) return;
-
+    setRunningTest(2);
     running = true;
-    _timer.reset();
     currentProgress.clear();
 
+    _timer = Stopwatch();
     _timer.start();
     _newIceSP.send('start');
 
     notifyListeners();
   }
 
-  Future<void> generateRandomNumbers() async {
+  Future<void> generateRandomNumbers(bool transferableTyped) async {
     if (running) return;
+    (transferableTyped) ? setRunningTest(1) : setRunningTest(0);
```

Comment:
I don't think we have a lint for this, but I *much* prefer avoiding ternary expressions as if they were statements and instead using if-thens for logic like this.